packages/core/backend/sirius-components-core/src/main/java/org/eclipse/sirius/components/core/api/IRepresentationMetadataProvider.java

```diff
@@ -21,6 +23,8 @@
  * @author pcdavid
  */
 public interface IRepresentationMetadataProvider {
-    boolean canHandle(String representationId);
-    RepresentationMetadata handle(String representationId);
+    default boolean canHandle(String representationId) {
+        return this.handle(representationId).isPresent();
+    }
+    Optional<RepresentationMetadata> handle(String representationId);
```

Comment:
We could probably rename the handle method since we don't have to follow our canHandle / handle pattern

Reply:
Done